easybuild/easyconfigs/n/NewHybrids/NewHybrids-1.1_Beta3-GCC-10.2.0.eb

```diff
@@ -0,0 +1,42 @@
+# Author: Denis Krišťák (INUITS)
+# License: GPLv2
+
+easyblock = 'Binary'
+
+name = 'NewHybrids'
+version = '1.1_Beta3'
+
+homepage = 'https://ib.berkeley.edu/labs/slatkin/eriq/software/software.htm#NewHybs'
+description = """This implements a Gibbs sampler to estimate the posterior probability that genetically sampled
+individuals fall into each of a set of user-defined hybrid categories."""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+
+extract_sources = True
+
+sources = [{
+    'filename': SOURCE_TAR_GZ,
+    'git_config': {
+        'url': 'https://github.com/eriqande',
+        'repo_name': 'newhybrids',
+        'commit': '6fc8fd9ca6e64ee0be08af12ec8bfe73a7de00cc',
+        'recursive': True,
+    }
+}]
+checksums = ['2cc1d3e95ce167312f3be971a3acd1951bf1cdd49313696a86ee22b7685fc56c']
+
+install_cmd = '%(builddir)s/newhybrids/Compile-with-no-gui-Linux.sh && '
+install_cmd += 'mkdir %(installdir)s/bin && '
+install_cmd += 'cp -a %(builddir)s/newhybrids/newhybrids-no-gui-linux.exe %(installdir)s/bin/newhybs && '
+install_cmd += 'cp -ar %(builddir)s/newhybrids/test_data %(installdir)s/'
+
+sanity_check_paths = {
+    'files': ['bin/newhybs'],
+    'dirs': [],
+}
+
+sanity_check_commands = [
```

Comment:
@boegel probably a trap here...when test data isnt found, it will inform you and await another input (name of a different test data file..)...which freezes sanity check because i use grep here (test run exits with code 1 so it wouldnt pass sanity check) and it just wont find it in output and there's no more new output...
when i add test_data to sanity_check_paths, it should tell you it's not there, but i dont think it will stop trying to execute sanity check cmds...